docs/azure/sdk/dependency-injection.md

```diff
@@ -110,6 +114,9 @@ In the [Register clients](#register-clients) section, you explicitly passed the
   },
   "Storage": {
     "ServiceUri": "https://mydemoaccount.storage.windows.net"
+  },
+  "ServiceBus": {
+    "NamespaceUri": "https://<your-namespace>.servicebus.windows.net"
```

Comment:
```suggestion
    "Namespace": "<your-namespace>.servicebus.windows.net"
```

This is not a URI and we should avoid calling it one.  The formal term is "fully qualified namespace" - which is a mouthful.  "Namespace" alone is probably the best short substitute.

The format shouldn't have any protocol prefix.